nichtparasoup.py

```diff
@@ -187,9 +186,12 @@ def cache_fill_loop():
                 key = crawler + "_" + site
 
                 if key not in info["images_per_site"] or info["images_per_site"][key] < min_cache_imgs_before_refill:
-                    while key not in info["images_per_site"] or info["images_per_site"][key] < min_cache_imgs:
```

Comment:
The crawler are now triggered once every time the Cache fill loop stops by and detects there are not enough images. If a crawler cant get more images, it is just tried again with the next loop. With the old implementation the crawler would crawl till enough images are in the cache, resulting in an endless loop.